pages/about.vue

```diff
@@ -90,57 +94,68 @@
             v-for="(cta, i) in callsToAction"
             :key="i"
           >
-            <!-- these should probably link to different places -->
             <div class="mb-4">
-              <Button v-if="cta.link" secondary nuxt :to="cta.link">{{ cta.button_text }}</Button>
+              <Button v-if="cta.link" secondary nuxt :to="cta.link">{{
+                cta.button_text
+              }}</Button>
               <Button v-if="!cta.link" secondary>
                 <a
-                  style="text-transform:none;"
+                  style="text-transform: none;"
```

Comment:
Shouldn't need this after the `Button` changes in https://github.com/covid19risk/website/pull/37/files